index.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  
+  <link rel="stylesheet" href="style.css"><title>Prickly Pear Seeds Oil</title>
+</head>
+<body>
+
+  <!-- the upper menu bar -->
+  <div class="background">
+
+
+    <div class="backgrounMenu">
+    <ul class="backgrounList">
+      <li><a href="#"><img src="images/facebook.png" alt="facebook-icon" ></a></li>
+      <li><a href="#"> <img id="tweeter" src="images/twitter (2).png"   alt="tweeter-icon"></a></li>
+      <li><a href="#">English</a></li>
+      <li><a href="#">My page</a></li>
+      <li><a href="#">logout</a></li>
+    </ul>
+    </div>
+  </div>
+
+
+<!-- the header -->
+
+
+  <header class="header">
+
+
+   <div class="menu">
+   
+    <div id="menuBar"><img src="images/menu.png" 
+      alt="menubar-image" ></div>
+     
+      <div class="logo"><a href="https://nessrine88.github.io/Capstone-Project/"><img src="images/White And Black Modern Abstract Beauty Logo.png"  alt="logo"> </a></div>
+      <div id="menuList">
+        <ul>
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li><a href="#"><span class="sp">Best offers</span></a></li>
+        </ul>
+      </div>
+    </div>
+    
+  </header>
+<main>
+  <section class="section1">
+  <h2></h2>
+  <h1 class="clipped-text"><span class="h1span">"Take care of your Beauty!"</span> <br> Welcome to Sultan-Oils Company</h1>
+  <p class="paraSection1">Discover the wonders of prickly pear seeds oil at our leading company. Experience the luxurious benefits of our premium skincare and wellness products enriched with essential fatty acids, antioxidants, and vitamins. We prioritize sustainability and ethical sourcing, making a positive impact on both your skin and the environment..</p>
+
+  <div class="event">25 July 2023</div>
+  <div >@Prickly Pear Seeds Oil Beauty Showcase in Tunisia: <br> the Essence of Natural Beauty</div>
```

Comment:
Kindly use the right font color for this text here 🔽 

![image](https://github.com/Nessrine88/Capstone-Project/assets/20537712/c03cc5b6-0ab5-4dd8-8c6b-734f90023d92)